code/__DEFINES/keybindings.dm

```diff
@@ -0,0 +1,184 @@
+GLOBAL_LIST_INIT(keybinding_validkeys, list(
+	"A",
+	"B",
+	"C",
+	"D",
+	"E",
+	"F",
+	"G",
+	"H",
+	"I",
+	"J",
+	"K",
+	"L",
+	"M",
+	"N",
+	"O",
+	"P",
+	"Q",
+	"R",
+	"S",
+	"T",
+	"U",
+	"V",
+	"W",
+	"X",
+	"Y",
+	"Z",
+	"Insert",
+	"Delete",
+	"Northwest",
+	"Southwest",
+	"Northeast",
+	"Southeast",
+	"F1",
+	"F2",
+	"F3",
+	"F4",
+	"F5",
+	"F6",
+	"F7",
+	"F8",
+	"F9",
+	"F10",
+	"F11",
+	"F12",
+	"0",
+	"1",
+	"2",
+	"3",
+	"4",
+	"5",
+	"6",
+	"7",
+	"8",
+	"9",
+	"Numpad1",
+	"Numpad2",
+	"Numpad3",
+	"Numpad4",
+	"Numpad5",
+	"Numpad6",
+	"Numpad7",
+	"Numpad8",
+	"Numpad9",
+	",",
+	".",
+	"&",
+	"é",
+	"\"",
+	"'",
+	"(",
+	"§",
+	"è",
```

Comment:
what the

Reply:
French layout uses these. Like stated in OP, this is really ugly. Any ideas on how to clean this up?